bot/rules/everyone_ping.py

```diff
@@ -0,0 +1,36 @@
+import random
+from typing import Dict, Iterable, List, Optional, Tuple
+
+from discord import Embed, Member, Message
+
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+
+async def apply(
+    last_message: Message,
+    recent_messages: List[Message],
+    config: Dict[str, int],
+) -> Optional[Tuple[str, Iterable[Member], Iterable[Message]]]:
+    """Detects if a user has sent an '@everyone' ping."""
+    relevant_messages = tuple(msg for msg in recent_messages if msg.author == last_message.author)
+
+    everyone_messages_count = 0
+    for msg in relevant_messages:
+        if "@everyone" in msg.content:
+            everyone_messages_count += 1
+
+    if everyone_messages_count > config["max"]:
+        # Send the user an embed giving them more info:
+        embed_text = f"Please don't try to ping {last_message.guild.member_count:,} people."
+
+        # Make embed:
+        embed = Embed(title=random.choice(NEGATIVE_REPLIES), description=embed_text, colour=Colours.soft_red)
+
+        # Send embed:
+        await last_message.channel.send(embed=embed)
```

Comment:
I do not believe this comment is necessary, it just repeats what the code says. Similar above with "Make embed".